python/thirdparty

```diff
@@ -0,0 +1 @@
+../cpp/thirdparty/
```

Comment:
This appears to fail the license check. I believe this is actually not needed anymore, but I'll confirm.